packages/visual-group/src/group-helper/group-utils.js

```diff
@@ -251,6 +255,9 @@ export const mutateAxesFromMap = (cacheMaps, axes) => {
     };
 };
 
+const hasPolarEncodings = layerConf => layerConf.mark === ARC || [RADIUS, ANGLE].some(field =>
+        getObjProp(layerConf.encoding, field));
```

Comment:
What if the layer config has both radius and x encoding, which will take precedence (no mark provided)